examples/LAPACK/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-LAPACK)
+
+hunter_add_package(LAPACK)
+
+string(COMPARE EQUAL "${LAPACK_LICENSES}" "" is_empty)
+if(is_empty)
+  message(FATAL_ERROR "Licenses not found")
+endif()
+
+message("LAPACK licenses:")
+foreach(x ${LAPACK_LICENSES})
+  message("* ${x}")
+  if(NOT EXISTS "${LAPACK_LICENSES}")
+    message(FATAL_ERROR "File not found")
+  endif()
+endforeach()
+
+find_package(LAPACK CONFIG REQUIRED)
+
+# Test double library creation
+find_package(LAPACK CONFIG REQUIRED)
+
+
+find_package(cblas CONFIG REQUIRED)
```

Comment:
the install of lapack (without CBLAS) provides only libraries, no headers. So there is no `lapack.h` to include.
To have an example with c-interface I added cblas. I'll move the current example to `examples/lapack/cblas` but I don't know how to create an example just using `liblapack.a` and `libblas.a`